### ISSUE TITLE
Build failure on Debian Unstable with 1.23.1 release

### ISSUE THREAD

**Issue by lukefromdc** [mate-desktop/atril#407]
#### Expected behaviour
Finished build, used these commands to build for packing into a .deb file:

`./autogen.sh  --prefix=/usr libdir=/usr/lib/x86_64-linux-gnu  --sysconfdir=/etc --enable-shared=yes --enable-static=no --disable-maintainer-mode  --disable-schemas-compile
`
` make -j`

#### Actual behaviour
make -j and make (second attempt w/o parallel processing) both failed with this error:
```

Warning: Could not merge el translation for msgid:
<firstname>MATE Documentation Project</firstname> <surname/> <affiliation><orgname>MATE Desktop</orgname></affiliation>
/bin/bash: line 3:  2080 Segmentation fault      itstool -m "${mo}" ${d}/C/index.docbook ${d}/C/legal.xml
make[2]: *** [Makefile:821: el/el.stamp] Error 139
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/luke/Desktop/Development/MATE_Development_Work/atril_1.23.1/help'
make[1]: *** [Makefile:670: all-recursive] Error 1
make[1]: Leaving directory '/home/luke/Desktop/Development/MATE_Development_Work/
```

#### MATE general version
1.23 from git master

#### Package version
1.23.1 as released 

#### Linux Distribution
Debian Unstable with locally compiled GTK 3.24.10 and glib 2.61.1

#### link to downstream report of your Distribution
None as this is a local build

**Comment by raveit65** [comment]
Did you build from released tarball or from git?
Still wondering that it doesn't fail to build at travis?
https://travis-ci.org/mate-desktop/atril/builds/575841047?utm_source=github_status&utm_medium=notification
And i did a `make distcheck` locally on fedora 30 before i pushed the new translations.
Usually this sounds like a format error in translations for the language.
For the moment you can disable the language for building as workaround.
https://github.com/mate-desktop/atril/blob/master/help/Makefile.am#L10
```
# Add linguas to be ignored, e.g. IGNORE_HELP_LINGUAS = ca de es fr
IGNORE_HELP_LINGUAS = el
```
and run autogen.sh again.

**Comment by lukefromdc** [comment]
Directly from git as I rarely download tarballs separately.

**Comment by sc0w** [comment]
here, over `mate-desktop 1.22.1` with `debian testing/bullseye` I can build and make `.deb` packages successfully, with `atril git master`.

**Comment by rbuj** [comment]
Opening and closing orgname tags mismatch in a Greek translation:
https://github.com/mate-desktop/atril/blob/fd995f8356cd587a410ea9043d78f03e153ad211/help/el/el.po#L179-L186

**Comment by lukefromdc** [comment]
Out of town now so can't test again. If you can't find it I will build with that language
disabled and be done with it. I know almost nothing about trasnlation stuff-and
have most locales purged for space

**Comment by raveit65** [comment]
Here with fedora i get only a compile warning.
Still wondering why compiling fails for you instead of printing a warning.
There are much more merge issues....
```
  GEN    el/
Warning: Could not merge el translation for msgid:
<firstname>MATE Documentation Project</firstname> <surname/> <affiliation><orgname>MATE Desktop</orgname></affiliation>
Warning: Could not merge el translation for msgid:
<imageobject><imagedata fileref="figures/atril_start_window.png" format="PNG"/> </imageobject> <textobject> <phrase>Shows Atril Document Viewer main window. Contains titlebar, menubar, toolbar and display area. Menubar contains File, Edit, View, Go and Help menus. </phrase> </textobject>
Warning: Could not merge el translation for msgid:
To resize the <application>Atril Document Viewer</application> window to have the same width and height as the screen, choose <menuchoice> <guimenu>View</guimenu> <guimenuitem>Full Screen</guimenuitem> </menuchoice>. To resize the <application>Atril Document Viewer</application> window to the original size, click on the <guibutton>Exit Full Screen</guibutton> button.
  GEN      ja/ja.mo
```
Any way , i will try to fix them at transifex.

**Comment by raveit65** [comment]
Fixed for master and 1.22.
https://github.com/mate-desktop/atril/commit/7a15121edafe66a60298c00d5da3e07ef9a0b938
https://github.com/mate-desktop/atril/commit/f39e4ec217cbee93bf5a2de0a70f639e511b2002
@lukefromdc 
This should solve your build failure, please test.

**Comment by lukefromdc** [comment]
Oh yeah, this fixed it, good to go
